tests/rbd/rbd_snap_clone_imported_image.py

```diff
@@ -0,0 +1,44 @@
+from tests.rbd.rbd_utils import Rbd
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(**kw):
+    """
+    Test case covered -
+    CEPH-9230 - verification snap and clone on an imported image
+    Pre-requisites :
+    1. Cluster must be up and running with capacity to create pool
+       (At least with 64 pgs)
+    2. We need atleast one client node with ceph-common package,
+       conf and keyring files
+
+    Test Case Flow:
+    1. Create a pool
+    2. Create a dummy file, import it as an image
+    3. Create snapshot of it and protect the snapshot
+    4. Create a clone
+    """
+    log.info("Running snap and clone operations on imported image")
+    rbd = Rbd(**kw)
```

Comment:
as per the test case configuration, no parameters are passed except test metadata. here we are using same to initialize Rbd class. is this needed?

Reply:
It was observed that its was required (example for ceph_nodes)